### DISCUSSION TITLE
Link homepage

### DISCUSSION THREAD

**Discussion by fabiobiasi** [globaleaks/globaleaks-whistleblowing-software#3858] [Q&A] [answered]
Salve,
ho installato Globaleaks e la sto usando per più siti, ho un problema sul link homepage, quando clicco sul logo mi riporta alla pagina iniziale e non a quella del sito specifico. Dove posso configurare correttamente l'homepage a cui rimandare? o in alternativa è possibile disattivare il link all'homepage?
Grazie

**Comment by evilaliv3** [comment, marked as answer]
Salve, può sperare meglio tramite alcuni esempi? cosa intende con sito specifico?

**Comment by fabiobiasi** [comment]
Grazie per la risposta,
io ho installato GlobaLeaks su un server Debian
la pagina principale la utilizzo come amministratore, ho successivamente creato dei siti per dare il servizio a più realtà
quindi ho una struttura fatta così
Pagina principale
-->sito1
-->sito2
-->sito3

Se mi posiziono, per esempio, nella pagina del sito 1 facendo click sul logo impostato in alto a sinistra mi aspetto che questo mi riporti all'home page del sito1 invece mi reindirizza nell'homepage della pagina principale.
La cosa strana è che se passo con la freccia del mouse sopra il logo il link riportato è corretto ma quando clicco mi riporta ugualmente alla pagina madre
Se vedi lo screen in basso è riportato il link del collegamento, che è corretto, ma se faccio click sul logo mi rimanda alla pagina madre.

![Screenshot 2023-12-11 123958](https://github.com/globaleaks/GlobaLeaks/assets/16292922/7e936403-d6c0-439e-9bda-28f5f80dfea0)

Spero di essermi spiegato correttamente.
Grazie
